conda-recipe/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "esgfpub" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/E3SM-Project/esgfpub.git
+  git_rev: v{{ version }}
+
+build:
+  noarch: python
+  script: "python -m pip install . -vv"
+
+about:
+  home: https://github.com/E3SM-Project/esgfpub
+  summary: Automated publication to ESGF
+
+requirements:
+  build:
+    - python >=3.8
+    - setuptools
+    - pip
+
+  run:
+    - python >=3.8
```

Comment:
Despite "build" and "run" being different, can they have competing dependencies?  Both say "python >= 3.8".